easybuild/easyconfigs/x/xmlf90/xmlf90-1.5.4-iimpi-2019b.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'xmlf90'
+version = '1.5.4'
+
+homepage = 'https://launchpad.net/xmlf90'
+description = """xmlf90 is a basic XML parsing library written in Fortran."""
+
+# toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+toolchain = {'name': 'iimpi', 'version': '2019b'}  # workaround to problem using module from Siesta
```

Comment:
@migueldiascosta What's this about?

Reply:
I suppose it's a old issue that popped up again in 2019b, something about Intel's mpiifort wrapper not recognizing modules built with GCCcore (also seen here I think, https://github.com/easybuilders/easybuild-easyconfigs/pull/9780#issuecomment-579214084) (?)

Reply:
Yes, ifort doesn't understand gfortran module files. They are incompatible.

Reply:
OK, then please clarify the comment, and just drop the commented out line?

Also, then it should be just `iccifort` rather than `iimpi`, no?